### DISCUSSION TITLE
Regenerate Report After Changing Parse Options

### DISCUSSION THREAD

**Discussion by prsf1233** [allinurl/goaccess#2366] [Q&A]
Hi, I have needed to exclude two IP addresses from my static report that runs from cron every hour.  I assumed that the report when run would have the data related to these two IP addresses removed but they still show up in the visits and hit counts. I do have disk persistence enabled, would I need to remove the database to have all the data refreshed?

Thanks

**Comment by allinurl** [comment]
Restoring an existing dataset from the db files does not parse the data but simply inserts it back into memory (it's simply a snapshot of what it was upon persisting it). I agree that it would be great to pre-process it and then store it, but performance would be a hit, for sure - maybe there should be an option?
